tests/test_optimizer.py

```diff
@@ -305,7 +316,11 @@ def test_optimizer(
 
     def run_trainer(batch_size: int, batch_time: float, client_mode: bool):
         nonlocal optimizer
-        model = nn.Linear(5, 1)
+        model = nn.Sequential(
```

Comment:
Please explain this change

Reply:
Resolved: this is likely due to the fact that powersgd needs 2+-dim tensors

Reply:
Just to make it clear, do you mean "tensors with >1 dimensions" or "tensors with each dimension having a size of >1"? Because I've tested the first one, and it appears to be true for a (5, 1) linear:

```
>>> a=torch.nn.Linear(5,1)
>>> a.weight.ndim
2
```

Reply:
PowerSGD requires one of the dimensions tobe greater than factorization rank. I added additional layer to create possible case to test the averager.